modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -281,48 +305,49 @@
 /datum/job/prisoner
 	alt_titles = list(
 		"Prisoner",
+		"Convict",
+		"Felon",
+		"Inmate",
```

Comment:
I don't actually have the authority for this one but I'd like a maintainer's input - can these actually be removed? Having the prisoners not all labelled consistently can _and has_ made it hard for security to find them in records, as the primary way of finding them is searching by job title.

Reply:
Yeah, I think that's a fair point. It's not like if it made much sense for them to not be called directly "Prisoner" in some capacity, since it's meant to be something that you're not in of your own accord, really.